spec/index.bs

```diff
@@ -8989,8 +9012,8 @@ operations are performed:
 <dl dfn-type=dict-member dfn-for=GPUStencilFaceState>
     : <dfn>compare</dfn>
     ::
-        The {{GPUCompareFunction}} used when testing fragments against
-        {{GPURenderPassDescriptor/depthStencilAttachment}} stencil values.
+        The {{GPUCompareFunction}} used when testing the {{RenderState/[[stencilReference]]}} value
+        against the fragment's {{GPURenderPassDescriptor/depthStencilAttachment}} stencil value.
```

Comment:
nit: seems unrelated, OK to include in this PR, but just a note in case you intended this for a different PR.

Reply:
Huh, yeah. That was meant for #4849. Not sure how it ended up here. Thanks for calling it out. (I mean, it's just a grammer/link fix either way, but I'll keep it with the intended PR.)